src/app/components/ArticlesList/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Article from '../../components/Article';
+
+import createUuidv4 from '../../utils/createUuidv4';
+
+import './scss/index.scss';
+
+const ArticlesList = (props) => {
+    const arrayOfNews = props.articles.map(article => (
+        <Article
+            key={createUuidv4()}
```

Comment:
The key shouldn't be generated randomly, please check it's mission in scope of reconcilliation.

Reply:
Done.
All keys have been changed to news url.